src/components/common/Accordion/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useState } from 'react';
+import { Box, BoxProps, Flex } from 'rebass/styled-components';
+
+import { Icon, Text } from '~components';
+import { ChevronDown } from '../Icon/library';
+
+export interface AccordionProps extends BoxProps {
+  title: string;
+}
+
+export const Accordion: FC<AccordionProps> = ({
+  title,
+  children,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Box
+      sx={{
+        borderBottomWidth: 1,
+        borderBottomStyle: 'solid',
+        borderBottomColor: 'grey600',
+      }}
+    >
+      <Flex
+        justifyContent="space-between"
+        alignItems="flex-start"
+        onClick={() => setOpen(!open)}
+        pb={1}
+        sx={{
+          cursor: 'pointer',
+        }}
+        {...props}
+      >
+        <Text variant="pmd" color="grey700" fontWeight="bold">
+          {title}
+        </Text>
+        <Icon
+          size={24}
+          sx={{
+            transform: open ? 'rotate(180deg)' : 'none',
+            transition: 'transform 0.2s ease-in-out',
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+          icon={<ChevronDown />}
+        />
+      </Flex>
+
+      <Box
+        sx={{
+          maxHeight: open ? '500px' : 0,
```

Comment:
Use https://streamich.github.io/react-use/?path=/story/sensors-usemeasure--docs and framer-motion to not limit the text by 500px